tests/test_wave_01.py

```diff
@@ -118,13 +118,12 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
 
-@pytest.mark.skip()
+    # Added assertions to test if correct movie was added to "watched"
```

Comment:
Good addition of the test. And for completeness, we could test the other attributes as well ("genre", "rating")